lib/services/ngsi/entities-NGSI-v2.js

```diff
@@ -352,7 +350,7 @@ function sendUpdateValueNgsi2(entityName, attributes, typeInformation, token, ca
         attributes,
         typeInformation
     );
-    let payload = {
+    const payload = {
```

Comment:
Before reviewing the PR, I kindly ask you if you could "rollback" the lint changes in this file?

That would remove lot of noise and reduce the scope of the PR

Thanks